tests/System.Buffers.Experimental.Tests/MemoryTests.cs

```diff
@@ -315,10 +314,15 @@ public override MemoryHandle Pin(int byteOffset = 0)
                 if (byteOffset != 0 && (((uint)byteOffset) - 1) / Unsafe.SizeOf<T>() >= _array.Length) throw new ArgumentOutOfRangeException(nameof(byteOffset));
```

Comment:
See earlier comments re: `Pin` method implementation: `if` checks, `Unsafe.Add<T>`, and parameter rename.